langchain/llms/vertexai.py

```diff
@@ -91,6 +106,25 @@ def validate_environment(cls, values: Dict) -> Dict:
             values["client"] = TextGenerationModel.from_pretrained(values["model_name"])
         return values
 
+    async def _acall(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[AsyncCallbackManagerForLLMRun] = None,
+    ) -> str:
+        """Call Vertex model to get predictions based on the prompt.
+
+        Args:
+            prompt: The prompt to pass into the model.
+            stop: A list of stop words (optional).
+            run_manager: A callback manager for async interaction with LLMs.
+
+        Returns:
+            The string generated by the model.
+        """
+        return await asyncio.wrap_future(
+                _VertexAICommon._get_task_executor().submit(self._predict, prompt, stop))
```

Comment:
is this just `self._get_task_executor()`?

Reply:
yes correct. I've changed it to match. I've also fixed lint errors.